webpack/ForemanTasks/Components/TasksTable/TasksTable.js

```diff
@@ -82,11 +87,19 @@ const TasksTable = ({
   };
 
   const taskActions = {
-    cancel: (id, name) => {
-      cancelTask(id, name, url, parentTaskID);
+    [CANCEL]: (id, name) => {
+      openClickedModal({
+        id,
+        name,
+        setModalOpen: modalProps.CANCEL.setModalOpen,
+      });
     },
-    resume: (id, name) => {
-      resumeTask(id, name, url, parentTaskID);
+    [RESUME]: (id, name) => {
+      openClickedModal({
+        id,
```

Comment:
`id` and `name` here refers to the task ID / task name, not the modal ID / modal name, correct?  If correct I think it would avoid confusion to name them `taskId` and `taskName`.

Reply:
Its for the task, changed, thanks